bin/load_db_scxa_marker_genes.sh

```diff
@@ -146,14 +144,19 @@ if [[ -z ${NUMBER_MGENES_FILES+x} || $NUMBER_MGENES_FILES -gt 0 ]]; then
 
   cat $markerGenesToLoad | awk -F',' 'BEGIN { OFS = "|"; } {print $1"_"$3"_"$4, $2, $5}' > ${groupMarkerGenesToLoad}.tmp
   
-  # Add in the markers from annotation sources
+  # Add in the markers from annotation source- basically match any non-numeric
+  # field in the file name
 
-  if [ -e "$inferredCelltypeMarkers" ]; then
-    tail -n +2 $inferredCelltypeMarkers | awk -F'\t' -v EXP_ID="$EXP_ID" 'BEGIN { OFS = "|"; } { gsub("^nan$","Not available",$1); print EXP_ID"_inferred cell type_"$1, $4, $8 }' >> ${groupMarkerGenesToLoad}.tmp
-  fi
-  if [ -e "$authorsInferredCelltypeMarkers" ]; then
-    tail -n +2 $authorsInferredCelltypeMarkers | awk -F'\t' -v EXP_ID="$EXP_ID" 'BEGIN { OFS = "|"; } { print EXP_ID"_authors inferred cell type_"$1, $4, $8 }' >> ${groupMarkerGenesToLoad}.tmp  
-  fi
+  re='^[0-9]+$'
+    
+  for markerGenesFile in $(ls $EXPERIMENT_MGENES_PATH/${EXP_ID}.marker_genes*.tsv); do
```

Comment:
Here you are reading all the marker genes_*inferred_cell_type.tsv associated with authors and ontology labels?

Reply:
Actually, anything matching the file pattern, just wanted it to be flexible.